tensorboard/webapp/metrics/store/metrics_selectors.ts

```diff
@@ -510,6 +510,13 @@ export const getMetricsCardMinMax = createSelector(
   }
```

Comment:
nit: can we update the doc above or the function name (getMinMaxStepFromCardState) to make it clear that
(1)this returns userMinMax if that exists, 
(2)otherwise it returns dataMinMax
(3) userMinMax is not necessary a subset of dataMinMax (think about example that dataMinMax=[-100,100] and userMinMax=[50.150])